modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/displayUndoDeleteSuccessToast.ts

```diff
@@ -62,4 +70,18 @@ export default function displayUndoDeleteSuccessToast(
 	};
 
 	openToast(openToastSuccessProps);
+
+	Liferay.componentReady(componentId).then(() => {
+		const toastLink = document.getElementById(componentId);
+
+		if (toastLink) {
+			toastLink.focus();
+		}
+	});
+
+	return {
+		dispose() {
+			Liferay.destroyComponent(componentId);
```

Comment:
I don't think this is necessary, the openToast util handles the destroying of the react component automatically when it is closed